pulsar-broker/src/main/java/org/apache/pulsar/broker/delayed/bucket/BucketDelayedDeliveryTracker.java

```diff
@@ -341,18 +339,24 @@ public synchronized boolean addMessage(long ledgerId, long entryId, long deliver
     private synchronized CompletableFuture<Void> asyncMergeBucketSnapshot() {
         List<ImmutableBucket> values = immutableBuckets.asMapOfRanges().values().stream().toList();
         long minNumberMessages = Long.MAX_VALUE;
+        long minScheduleTimestampSum = Long.MAX_VALUE;
```

Comment:
Is variable `minScheduleTimestampSum` unnecessary?

Reply:
It is used in line-356

Reply:
Ah, it helps to find the first segment, right?

Reply:
Oh, it is used to find the bucket which the min scheduled time of the next segment to merge, avoid preload indexes with long not expired.